geoportailv3/templates/index.html

```diff
@@ -439,7 +439,7 @@ <h2 translate>Feedback</h2>
          appModule.value('ngeoScaleselectorTemplateUrl', 'infobar/scaleselector.html');
          appModule.value('ngeoOfflineTestUrl', 'infobar/scaleselector.html');
          appModule.value('ngeoOfflineTemplateUrl', function() {
-           return '${request.static_url('%s/ngeo/src/offline/component.html' % node_modules_path)}'});
+           return '${request.static_url('geoportailv3:static/js/offlineNgeoComponent.html')}'});
```

Comment:
You should not construct a static_url path but instead simply use the `offlineNgeoComponent.html` string.

Reply:
(that way it will use the template cache). You should also check everything is working as expected by using the browser offline mode. You will need to re-enable the appcache for local dev to be able to test.